webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsActions.js

```diff
@@ -0,0 +1,13 @@
+import { API_OPERATIONS, get } from 'foremanReact/redux/API';
+import { foremanApi } from '../../../../../services/api';
+import { MODULE_STREAMS_KEY } from './ModuleStreamsConstants';
+
+
+export const getHostModuleStreams = (hostId, params) => get({
+  type: API_OPERATIONS.GET,
+  key: MODULE_STREAMS_KEY,
+  url: foremanApi.getApiUrl(`/hosts/${hostId}/module_streams`),
+  params,
```

Comment:
Not sure how you guys are handling errors in the hosts pages, but if not otherwise handled I would suggest adding the errorToast to this in case of an issue.